joke-data-crawler/src/s3_upload_joke.py

```diff
@@ -0,0 +1,10 @@
+import json
+import boto3
+
+s3_client = boto3.client("s3")
```

Comment:
do you use this for testing?

In lambda this function will not work

Reply:
you can  only save to temp in lambda

Reply:
upload ist directyl called in lambda

Reply:
yes just for testing. I am pushing the real codes soon